.github/workflows/cargo.yml

```diff
@@ -54,5 +47,9 @@ jobs:
           override: true
       - name: Check out ci repo
         run: git clone https://github.com/stratis-storage/ci.git
-      - name: Run ${{ matrix.task }}
-        run: ${{ matrix.task }}
+      - name: Run comparisons of version specs with available Fedora packages
+        # yamllint disable rule:line-length
+        run: |
+          COMPARE_FEDORA_VERSIONS=./ci/dependency_management/compare_fedora_versions FEDORA_RELEASE=rawhide IGNORE_ARGS="--ignore-category low" make -f Makefile_dependencies check-fedora-versions
+          COMPARE_FEDORA_VERSIONS=./ci/dependency_management/compare_fedora_versions FEDORA_RELEASE=f36 IGNORE_ARGS="--ignore-category low --ignore-high pretty-hex" make -f Makefile_dependencies check-fedora-versions
```

Comment:
Plz swap the positions of the f37 and f36 release lines. Also, remove the '--ignore-high pretty-hex' everywhere. Do this for both 'cargo.yml' and 'nightly.yml'.

Reply:
Updated via amend.  For nightly.yml, I removed the `IGNORE_ARGS` variable, since it no longer has any arguments assigned.